Jenkinsfile

```diff
@@ -83,4 +82,35 @@ demos['databound'] = {
     }
 }
 
-parallel(demos)
+parallel(demos)*/
+
+node('docker') {
+    infra.withDockerCredentials {
+        def image
+        def imageName = "${env.DOCKERHUB_ORGANISATION}/jenkinsfile-runner"
+        def imageTag
+        def branchName = currentBuild.projectName
+
+        stage('Build container') {
+            timestamps {
+                deleteDir()
+                def scmVars = checkout scm
+
+                def shortCommit = scmVars.GIT_COMMIT
+                imageTag = branchName.equals("master") ? "latest" : branchName
+                echo "Creating the container ${imageName}:${imageTag}"
+                image = docker.build("${imageName}:${imageTag}", '--no-cache --rm .')
+            }
+        }
+
+        // TODO: Update when PR-99 is merged
+        if (branchName.startsWith('master') || branchName.startsWith('PR-99')) {    
```

Comment:
One doubt about this (sorry for not bringing it up before): what will happen with tags? Because I guess we want it published?

Reply:
Since it's for experimental images, we only care about master.